menu_delivery/menu_delivery.py

```diff
@@ -0,0 +1,246 @@
+from __future__ import print_function, unicode_literals
+from datetime import datetime
+import json
+import sys
+
+from PyInquirer import style_from_dict, Token, prompt, Separator
+from PyInquirer import Validator, ValidationError
+from pyfiglet import Figlet
+
+from p2pnode import P2PNode
+import regex
+
+
+class initialization(P2PNode):
+    """Initializes the Program and creates either a server or client instance.
+    For servers, it also creates the CLI interface for updates from the client."""
+
+    def __init__(self, node_type):
+        self.node_type = node_type
+
+    def initialize_node(self):
+        """Initializes and starts the node by taking the node_type
+            from the class variable. It also creates the CLI interface
+            for updates from the client.
+
+        Returns:
+            node: client or server node, which is a Node Object from p2pnode
+        """
+        if self.node_type["Node_Type"] == "Client":
+            client = P2PNode("127.0.0.1", 10001, 1)
+            print("Client node initialized")
+            return client
+
+        server = P2PNode("127.0.0.1", 10002, 2)
+        server.debug = True
+        server.server_table.add_column("Order Number")
+        server.server_table.add_column("Order Description")
+        print("Server node initialized")
+        server.start()
+        return server
+
+
+class CLI:
+    """Creates multiple Command Line Interface, starting from connecting to
+    nodes and selecting menus.
+    """
+
+    def __init__(self):
+        """Styles the Command Line Interface, and adds the Title of the CLI
+        Imports the menu options from a json file.
+        """
+        self.style = style_from_dict(
+            {
+                Token.Separator: "#cc5454",
+                Token.QuestionMark: "#673ab7 bold",
+                Token.Selected: "#cc5454",  # default
+                Token.Pointer: "#673ab7 bold",
+                Token.Instruction: "",  # default
+                Token.Answer: "#f44336 bold",
+                Token.Question: "",
+            }
+        )
+        f = Figlet(font="slant")
+        print(f.renderText("Menu Delivery v0.1"))
+        self.json = "./menu.json"
+        self.menu = self.import_menu_from_json()
+
+    def import_menu_from_json(self):
+        """Imports the menu from the user editable json file and
+        adds it as a dictionary for the CLI to implement.
+
+        Returns:
+            menu: dictionary of menus converted from the json file.
+        """
+        with open(self.json) as json_file:
+            menu = json.load(json_file)
+        return menu
+
+    def add_choices(self, questions, category):
```

Comment:
doesnt need to be a member function here, perhaps a helper function file/section would make this cleaner